lib/phoenix/live_dashboard/live/metrics_live.ex

```diff
@@ -79,7 +79,6 @@ defmodule Phoenix.LiveDashboard.MetricsLive do
   end
 
   def handle_info({:node_redirect, node}, socket) do
-    args = if group = socket.assigns.group, do: [group], else: []
-    {:noreply, push_redirect(socket, to: live_dashboard_path(socket, :metrics, node, args))}
+    {:noreply, push_redirect(socket, to: live_dashboard_path(socket, :home, node))}
```

Comment:
I understand why you did these changes but I am not sure if we should really do it. This means that for the most common case (which are all nodes running Phoenix), the user experience got worst. Maybe we should keep this as is and then on the `metrics_live.mount/3`, we check if the dashboard is running and if not we redirect to home?

Reply:
I copied the behaviour from `os_mon_live`. It's not ideal, but I'm not that familiar with the codebase yet, so want to keep my changes to a minimum. I guess we need to change it 3 places (os mon, request logger and metrics). Can do it now, or perhaps in a later PR? 

Perhaps it's an idea to assign :capabilities in the top level assigns, and use them both in the menu and on mounts where necessary?